NOTICE.md

```diff
@@ -51,3 +53,15 @@ android:
  * android (Apache license)
  * org.mapsforge, LGPL
  * VTM, LGPL
+
+## Data
+
+|source | license | used as default | included in repo |
+|---------|-----------|---------|------|
+|OpenStreetMap data for the road network | [ODBL](https://www.openstreetmap.org/copyright) | yes | yes
+| GTFS NRW | TODO link | no | yes (TODO - is it? I couldn't find it?)
+| SRTM elevation | [public domain](https://www2.jpl.nasa.gov/srtm/), [acknowledgement](https://lpdaac.usgs.gov/citing_our_data) | no | yes
+| CGIAR elevation | [allowed usage for GraphHopper](https://graphhopper.com/public/license/CGIAR.txt) | no | yes (TODO - really?, where?)
```

Comment:
I thought in test, but then probably not :) Will have a look

Reply:
We seem to use just SRTM data for tests under core/files

Reply:
Yes, I think so too :).